ui-tests/cypress/integration/pages/management.create.spec.js

```diff
@@ -40,7 +40,7 @@ describe('Pages Management - Create', () => {
 
         cy.wait(2000);
         // page should be created successfully
-        currentPage.getContent().getRowByPageName(`${template} - test en`).should('be.visible');
+        currentPage.getContent().getTableRows().contains(`${template} - test en`).should('be.visible');
```

Comment:
This kind of assertion is going to fail weirdly if the element is not present, it is better to assert `tableRows.should('contain', ${template} - test en)`